resolutionScaleFiller.py

```diff
@@ -268,7 +269,7 @@ def main():
         ntuple = HGCalNtuple(opt.fileString)
         eventLoop(ntuple, refName, objName, gun_type, pidSelected, GEN_engpt, histDict)
 
-    f = ROOT.TFile("{}_{}_{}GeV_{}_{}_{}.root".format(gun_type, pidSelected, GEN_engpt, refName, objName, tag), "recreate")
```

Comment:
Hi @ingorbunov - doesn't it make more sense to keep the formatting like that, but add a parameter that can then be set by the batch script?